### PR TITLE
Improve index and contributing docs by fixing some typos, links and more comprehensive English

### PR DESCRIPTION
**Pull request by raulcd**
I was going to read the documentation in order to start contributing and I saw there was some typos, a broken link to ```_GitHub``` instead of ```GitHub_``` and some English was difficult to understand.

I hope it makes sense. If you want me I can review the rest of the documentation for typos/improvements.

### CODE REVIEW

**Comment by kxepal** [comment]
Great! Thank you (: Please, check the rest of the documentation. Such kind of help is very appreciated.

**Comment by raulcd** [comment]
@kxepal Do you want me to do it on this PR? Or you want to merge this one?

**Comment by kxepal** [comment]
I think that one PR that handles this problem is fine. Will fix all the typos with the single merge (:

File: docs/contributing.rst [hunk 1]
```diff
@@ -74,35 +75,37 @@ Any extra texts (print statements and so on) should be removed.
 Tests coverage
 --------------
 
-We are strongly keeping our test coverage, please don't make it worse.
+We are strongly trying to have a good test coverage, 
+please don't make it worst.
```
Comment (benjamin-bader) [inline docs/contributing.rst:79]: This would read more naturally as "We are trying hard to have good test coverage; please don't make it worse."

Comment (raulcd) [inline docs/contributing.rst:79]: Thanks, I'll modify it. It was just polishing a little bit.

**Comment by asvetlov** [comment]
Thanks very much, Raul!

**Pull request merged**